packages/wordpress/components/blockStyles/blocks/button.js

```diff
@@ -0,0 +1,25 @@
+import { createGlobalStyle } from 'styled-components';
+import { siteTheme } from '@irvingjs/styled/utils';
+
+/* eslint-disable indent */
+export const ButtonBlock = createGlobalStyle`
+  .wp-block-button__link {
+    border: ${siteTheme('blocks.button.border', '12x solid blue')};
+    border-radius: ${siteTheme('blocks.button.borderRadisu', 0)};
```

Comment:
`12px` seems rather large :) did you mean `1px`?

```suggestion
    border: ${siteTheme('blocks.button.border', '1px solid blue')};
    border-radius: ${siteTheme('blocks.button.borderRadius', 0)};
```